### PR TITLE
Add ability to see all in otherHoldings

### PR DESCRIPTION
**Pull request by createthis**
Add ability to see HUSH in otherHoldings #22

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -116,4 +116,6 @@ typings/
 settings.json
 yarn.lock
 dist/
+*.swp
+**/*.swp
```
Comment (barryvdh) [inline .gitignore]: This doesn't belong in the commit

**Comment by barryvdh** [comment]
Makes sense to allow small coins I guess, otherwise we only have the top100.

**Comment by benmarten** [comment]
Used in #25
